src/commands/build.rs

```diff
@@ -1,16 +1,109 @@
 pub async fn build(config: &fpm::Config) -> fpm::Result<()> {
     tokio::fs::create_dir_all(config.build_dir()).await?;
 
-    let (documents, config) = fpm::get_documents_with_config(config).await?;
+    match (
+        config.package.translation_of.as_ref(),
+        config.package.translations.is_empty(),
+    ) {
+        (Some(_), true) => {
+            // No package can be both a translation of something and has its own
+            // translations, when building `config` we ensured this was rejected
+            unreachable!()
```

Comment:
Maybe we should model `Package` as an `enum`, `Simple`, `TranslationOf`, and `Translated` to make this match clause simpler and not having to deal with such "impossible" cases.